src/explore-education-statistics-admin/src/config/ckEditorConfig.ts

```diff
@@ -0,0 +1,104 @@
+import { HeadingOption, ResizeOption } from '@admin/types/ckeditor';
+
+export const toolbarConfigs = {
+  full: [
+    'heading',
+    '|',
+    'bold',
+    'italic',
+    'link',
+    '|',
+    'bulletedList',
+    'numberedList',
+    '|',
+    'blockQuote',
+    'insertTable',
+    'imageUpload',
+    '|',
+    'redo',
+    'undo',
+    '|',
+    'comment',
+  ],
+  simple: [
+    'bold',
+    'italic',
+    'link',
+    '|',
+    'bulletedList',
+    'numberedList',
+    '|',
+    'redo',
+    'undo',
+  ],
+};
+
+export const defaultAllowedHeadings = ['h3', 'h4', 'h5'];
+
+export const headingOptions: HeadingOption[] = [
+  {
+    model: 'heading1',
+    view: 'h1',
+    title: 'Heading 1',
+    class: 'ck-heading_heading1',
+  },
+  {
+    model: 'heading2',
+    view: 'h2',
+    title: 'Heading 2',
+    class: 'ck-heading_heading2',
+  },
+  {
+    model: 'heading3',
+    view: 'h3',
+    title: 'Heading 3',
+    class: 'ck-heading_heading3',
+  },
+  {
+    model: 'heading4',
+    view: 'h4',
+    title: 'Heading 4',
+    class: 'ck-heading_heading4',
+  },
+  {
+    model: 'heading5',
+    view: 'h5',
+    title: 'Heading 5',
+    class: 'ck-heading_heading5',
+  },
+];
+
+export const imageToolbar: string[] = [
```

Comment:
Don't think we need to specify `string[]` here. Typescript should be able to infer the type.